reference/channels.v3.yml

```diff
@@ -116,12 +116,42 @@ paths:
           $ref: '#/responses/error_Resp'
       description: Returns a list of *Channels*. Will always return the BigCommerce storefront with an ID of `1`.
```

Comment:
This is not something that you were supposed to do, but I'm realizing we could improve as I read this, but let's change this to "will always return the default BigCommerce storefront with an ID of `1`". The default storefront is always the first one that is created when a BigCommerce store is provisioned. We may want to add that in there, but I'll let you wordsmith that :)

Reply:
@amckemie Submitted changes on [PR 111](https://github.com/bigcommerce/api-specs/pull/111).